src/config.ts

```diff
@@ -13,6 +13,10 @@ export default {
     AsapPubKeyBaseUrl: process.env.ASAP_PUB_KEY_BASE_URL,
     AsapJwtAcceptedAud: process.env.ASAP_JWT_AUD,
     AsapJwtAcceptedHookIss: process.env.ASAP_JWT_ACCEPTED_HOOK_ISS,
+    CloudProvider: process.env.CLOUD_PROVIDER || 'oracle',
+    InstanceConfigurationId:
+        process.env.INSTANCE_CONFIGURATION_ID ||
+        'ocid1.instanceconfiguration.oc1.phx.aaaaaaaaued7h55rvp7dwp3hzcgck6ropvycpgxtxcpm6ljwih3e4yw6r4aq',
```

Comment:
We shouldn't hardcode anything here, but instead put any development options into a .env file in the same directory.

Reply:
Done, PTAL